### PR TITLE
Update naming conventions

### PR DESCRIPTION
**Pull request by lnagle**

### CODE REVIEW

File: lib/ddq-backend-mock.js [hunk 1]
```diff
@@ -212,23 +212,28 @@ module.exports = (crypto, EventEmitter, fs, timers) => {
 
 
         /**
-         * Hey Listen!
-         * Initiates the polling.
+         * Initiates the listening.
          */
-        listen() {
+        open() {
```
Comment (fidian) [inline lib/ddq-backend-mock.js]: Why does `open()` exist?

**Review by fidian — CHANGES_REQUESTED** [review]
It seems that `open()` is a helper method to simply start polling.  That should either be logic handled by the ddq core or eliminated entirely.

**Comment by GoogilyBoogily** [comment]
The version bump isn't correct for what's being changed.

File: package.json [hunk 2]
```diff
@@ -27,5 +27,5 @@
     "scripts": {
         "test": "eslint ."
     },
-    "version": "1.0.4"
+    "version": "1.0.5"
```
Comment (GoogilyBoogily) [inline package.json]: The changes aren't backwards compatible, so the version bump isn't correct.

File: lib/ddq-backend-mock.js (hunk 1 above)
Comment (lnagle) [reply lib/ddq-backend-mock.js]: An oversight. Removed.

File: package.json (hunk 2 above)
Comment (lnagle) [reply package.json]: Fixed.

**Comment by AbsentSemicolon** [comment]
Looks good to me.

**Comment by GoogilyBoogily** [comment]
Looks good to me also.

**Review by fidian — APPROVED** [review]

**Pull request merged**